index.md

```diff
@@ -21,17 +21,31 @@ about/projects
 :caption: Administrator Guide
 :maxdepth: 1
 
-admin/support
 admin/environment
-admin/interfaces
-admin/data
-admin/users
 admin/migrate
 admin/content
 ```
 
+```{toctree}
+:caption: Administrator configuration guides
```

Comment:
What is the relationship between the configuration guides and the how-to guides? They seem similar to me so trying to figure out why they have two different top-level sections

Reply:
That's a good question. I was thinking of configuration guide as a list of all possible options, but that can probably be better written as a how-to guide as well. We can have a separate administrator's reference if need be. I'll try to rewrite them.

Reply:
@choldgraf I think configuration options require 2i2c engineer support and how-to guides do not. I've clarified that with subheadings here.